core/common/src/main/java/alluxio/Constants.java

```diff
@@ -160,6 +161,12 @@
   public static final String UNDERFS_S3_ADMIN_THREADS_MAX = "alluxio.underfs.s3.admin.threads.max";
   public static final String UNDERFS_S3_UPLOAD_THREADS_MAX =
       "alluxio.underfs.s3.upload.threads.max";
+  public static final String UNDERFS_S3A_SECURE_HTTP_ENABLED =
+      "alluxio.underfs.s3a.secure.http.enabled";
+  public static final String UNDERFS_S3A_SERVER_SIDE_ENCRYPTION_ENABLED =
```

Comment:
Currently SERVER_SIDE_ENCRYPTION is only supported for s3a. But we would probably also support SSE for s3n in the future...
A more general question for our configuration naming conversion, between _S3A_ and _S3_ ?